settings.gradle

```diff
@@ -76,6 +76,15 @@ List projects = [
   'qa:wildfly'
 ]
 
+File examplePluginsDir = new File(rootProject.projectDir, 'plugins/examples')
```

Comment:
> I really want the names to line up if I can have it.

The names already don't line up, see for example the core hack we have around the same spot in settings.gradle. I really dont' see why the directory name matching the project name matters. These aren't things we ever have to deal with, other than in gradle, and gradle doesn't care what the names are, only that there is a dir associated with each project. `cd`ing into the `plugins/examples` dir still allows running tests as normal.